axelor-base/src/main/resources/views/ImportExportTranslation.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_6.1.xsd">
+
+  <grid name="import-export-translation-grid" title="Import/Export translation"
+    model="com.axelor.apps.base.db.ImportExportTranslation">
+    <field name="name" title="Name"/>
+    <field name="languageSet" title="Languages" widget="tagSelect"/>
+    <field name="uploadFile" title="Uploaded File"/>
+
+  </grid>
+
+  <form name="import-export-translation-form" title="Import/Export translation"
+    model="com.axelor.apps.base.db.ImportExportTranslation">
+    <panel name="infoPanel" title="info" showTitle="false" colSpan="12">
+      <field name="name" title="Name" required="true"/>
+      <field name="languageSet" title="Languages" widget="tagSelect" required="true"
+        domain="self.isNative = false"/>
+    </panel>
+    <panel name="importExportPanel" title="Import/Export" showTitle="false" colSpan="12">
+      <button name="export" title="Export" colSpan="4"
+        onClick="save,action-import-export-translation-method-export"/>
+      <field name="uploadFile" title="Uploaded File" colSpan="4" widget="binary-link"/>
+      <button name="import" title="Import" colSpan="4"
+        onClick="save,action-import-export-translation-method-import"
+        readonlyIf="uploadFile == null"/>
+    </panel>
+    <panel name="importExportHistoryPanel" title="Import/Export Histoty" showTitle="false"
+      colSpan="12">
+      <field name="importExportTranslationHistoryList"
```

Comment:
remove the previous panel and use a panel-related for importExportTranslationHistoryList set it in readonly